CONTRIBUTING.md

```diff
@@ -77,25 +77,23 @@ To contribute a patch, the workflow is a as follows:
 2. Create topic branch
 3. Commit patches
 
-In general commits should be atomic and diffs should be easy to read. For this
-reason do not mix any formatting fixes or code moves with actual code changes.
+Please keep commits should atomic and diffs easy to read. For this reason
+do not mix any formatting fixes or code moves with actual code changes.
 Further, each commit, individually, should compile and pass tests, in order to
 ensure git bisect and other automated tools function properly.
 
-When adding a new feature thought must be given to the long term technical debt.
-Every new features should be covered by unit tests.
+Please cover every new features with unit tests.
 
 When refactoring, structure your PR to make it easy to review and don't hesitate
 to split it into multiple small, focused PRs.
 
 Commits should cover both the issue fixed and the solution's rationale.
-These [guidelines](https://chris.beams.io/posts/git-commit/) should be kept in
-mind.
+Please keep these [guidelines](https://chris.beams.io/posts/git-commit/) in mind.
 
 To facilitate communication with other contributors, the project is making use
 of GitHub's "assignee" field. First check that no one is assigned and then
 comment suggesting that you're working on it. If someone is already assigned,
-don't hesitate to ask if the assigned party or previous commenters are still
+don't hesitate to ask if the assigned party or previous commentators are still
```

Comment:
In d581c9b546fb5b049aea7ba27f24edd1c44f811a:

I think `commenters` was correct.